src/Management/src/Abstractions/Configuration/EndpointOptions.cs

```diff
@@ -45,14 +45,18 @@ public virtual string? Path
     /// <summary>
     /// Gets the list of HTTP verbs that are allowed for this endpoint.
     /// </summary>
-    public IList<string> AllowedVerbs { get; }
+    public IList<string> AllowedVerbs { get; private set; } = new List<string>();
 
-    protected EndpointOptions()
+    internal HashSet<string> GetSafeAllowedVerbs()
+    {
+        // Caution: Mapping with an empty string in the list results in exposing the endpoint at ALL verbs.
```

Comment:
There's a test below that finds NOT FOUND for GET and POST after setting `allowedVerbs` to be an empty string, is this comment correct?

Reply:
Not sure what you mean by that, can you clarify?

Reply:
This comment says  `Caution: Mapping with an empty string in the list results in exposing the endpoint at ALL verbs.`
But the test `...Actuators.Refresh.HttpVerbInConventionalRoutingTest.Can_be_configured_to_allow_no_verbs` uses an empty string for `AllowedVerbs` and then asserts the response is NotFound

Reply:
`GetSafeAllowedVerbs` is called by the mapper, so that the test works. A user configures an empty string, which must result in not exposing the endpoints, instead of exposing it on all verbs.

Reply:
Ohhh. I was misunderstanding the purpose of this comment. It's explaining why we have `.Where(verb => verb.Length > 0)`, not describing bad ideas for configuration

Reply:
The comment explains _why_ this method does what it does.